Editor/BacktraceConfigurationEditor.cs

```diff
@@ -1,4 +1,5 @@
 using Backtrace.Unity.Model;
+using Backtrace.Unity.Model.Breadcrumbs;
```

Comment:
Why do we need `Breadcrumbs` include now?

Reply:
This change needs to analyse possible labels available in multi select dropdown by using reflection mechanism on enum type. Since we're displaying options for enum we need to include this using to start using it in the reflection code.

Reply:
Cool makes sense, thank you!